models/task/task.go

```diff
@@ -62,27 +63,28 @@ type Task struct {
 
 // DBModel is the "strict" representation of a task in DB, as expressed in SQL schema
 type DBModel struct {
-	ID                int64     `json:"-" db:"id"`
-	PublicID          string    `json:"id" db:"public_id"`
-	Title             string    `json:"title" db:"title"`
-	TemplateID        int64     `json:"-" db:"id_template"`
-	BatchID           *int64    `json:"-" db:"id_batch"`
-	RequesterUsername string    `json:"requester_username" db:"requester_username"`
-	WatcherUsernames  []string  `json:"watcher_usernames,omitempty" db:"watcher_usernames"`
-	ResolverUsernames []string  `json:"resolver_usernames,omitempty" db:"resolver_usernames"`
-	Created           time.Time `json:"created" db:"created"`
-	State             string    `json:"state" db:"state"`
-	StepsDone         int       `json:"steps_done" db:"steps_done"`
-	StepsTotal        int       `json:"steps_total" db:"steps_total"`
-	LastActivity      time.Time `json:"last_activity" db:"last_activity"`
+	ID                int64             `json:"-" db:"id"`
+	PublicID          string            `json:"id" db:"public_id"`
+	Title             string            `json:"title" db:"title"`
+	TemplateID        int64             `json:"-" db:"id_template"`
+	BatchID           *int64            `json:"-" db:"id_batch"`
+	RequesterUsername string            `json:"requester_username" db:"requester_username"`
+	WatcherUsernames  []string          `json:"watcher_usernames,omitempty" db:"watcher_usernames"`
+	ResolverUsernames []string          `json:"resolver_usernames,omitempty" db:"resolver_usernames"`
+	Created           time.Time         `json:"created" db:"created"`
+	State             string            `json:"state" db:"state"`
+	StepsDone         int               `json:"steps_done" db:"steps_done"`
+	StepsTotal        int               `json:"steps_total" db:"steps_total"`
+	LastActivity      time.Time         `json:"last_activity" db:"last_activity"`
+	Tags              map[string]string `json:"tags,omitempty" db:"tags"`
 
 	CryptKey        []byte `json:"-" db:"crypt_key"` // key for encrypting steps (itself encrypted with master key)
```

Comment:
Unrelated but this should probably disappear, something to look into @rbeuque74 ?